### DISCUSSION TITLE
How to import json modules in webdriverio 8 ?

### DISCUSSION THREAD

**Discussion by syamphaneendra** [webdriverio/webdriverio#10233] [Community]
How to import json modules in webdriverio 8 ? Its not allowing to import json modules with webdriverio 8 integration with typescript.

**Comment by christian-bromann** [comment]
this is not related to WebdriverIO, there are many tutorials you can look up that explain how to set this up: https://reacthustle.com/blog/how-to-import-a-json-file-in-typescript
